src/main/java/com/iota/iri/service/snapshot/impl/SnapshotServiceImpl.java

```diff
@@ -28,15 +29,16 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
 
 /**
- * Implements the basic contract of the {@link LocalSnapshotService}.
+ * Implements the basic contract of the {@link SnapshotService}.
  */
-public class LocalSnapshotServiceImpl implements LocalSnapshotService {
+public class SnapshotServiceImpl implements SnapshotService {
     /**
      * Logger for this class allowing us to dump debug and status messages.
      */
-    private static final Logger log = LoggerFactory.getLogger(LocalSnapshotServiceImpl.class);
+    private static final Logger log = LoggerFactory.getLogger(SnapshotServiceImpl.class);
```

Comment:
for example here it is lower case (like the rest of the project which is awesome)